idle_sweep_test.go

```diff
@@ -280,3 +282,64 @@ func TestIdleSweepMisconfiguration(t *testing.T) {
 	assert.Nil(t, ch, "NewChannel should not return a channel")
 	assert.Error(t, err, "NewChannel should fail")
 }
+
+func TestIdleSweepIgnoresConnectionsWithCalls(t *testing.T) {
+	ctx, cancel := NewContext(time.Second)
+	defer cancel()
+
+	clientTicker := testutils.NewFakeTicker()
+	clock := testutils.NewStubClock(time.Now())
+
+	listener := newPeerStatusListener()
+	serverOpts := testutils.NewOpts().
+		AddLogFilter("Skip closing idle Connection as it has pending calls.", 1).
```

Comment:
Was looking over this and I initially thought that it would check and verify for this log message. It'd probably be great if we could do that as well.

Reply:
Added a TODO